modules/mod_seo_sitemap/models/m_seo_sitemap.erl

```diff
@@ -164,7 +168,7 @@ maybe_add_category_attrs(#{ category_id := CatId } = Map, Context) ->
                 undefined -> <<"weekly">>;
                 CF -> CF
             end,
-            Map1#{ changefreq := Freq };
+            Map1#{ changefreq => Freq };
```

Comment:
Good side-catch 😬

Reply:
Weird that this seemed to work, and compiled...